examples/nextjs-shadcn/src/app/embeds.tsx

```diff
@@ -71,6 +76,9 @@ export function Embeds(props: { embeds: Array<Embed> }) {
         wallet: {
           address,
         },
+        farcaster: {
+          fid: "1214",
```

Comment:
hm will this be a problem in prod? can we easily find connected users fid

Reply:
I think it’s a fair assumption that clients should be able to provide the user’s FID, but we should explicitly handle the case where the FID isn’t available. Should only affect the holder info

if you’re referring to the example repo specifically then we can check if neynar can provide a reverse lookup for an fid given an address